app/Models/User.php

```diff
@@ -71,6 +71,12 @@ public function routeNotificationForPushover()
         return data_get($this->settings, 'notifications.pushover.user_key');
     }
 
+    public function routeNotificationForGotify()
+    {
+        $settings = data_get($this->settings, 'notifications.gotify');
+        return $settings && isset($settings['url'], $settings['token']) ? $settings : false;
```

Comment:
I think this will break coding standards, need a new line before any return statement

To fix coding standards `lando phpcs-fix` to also check static code analysis `lando phpcs`